### ISSUE TITLE
allow to use aria2 witout daemon

### ISSUE THREAD

**Issue by llaumgui** [DjazzLab/ocdownloader#32]
Hi,

currently daemon aria2 is mandatory. Is it possible to allow to use simple aria2 command without daemon ?

**Comment by DjazzLab** [comment]
Yep ocDownloader will work like this (like the current cURL download engine, without the use of a daemon) in a future release.
Because I'm working on a post-processing actions system, It makes more sense to run a simple aria2c command first and trigger the post-processing action after.

**Comment by llaumgui** [comment]
It's not a "question" but a feature request. Why not tag "enhancement" ? Especially if it's planned...

**Comment by DjazzLab** [comment]
I tagged this issue as a question because you asked me a question.
And sure you're right, I add the enhancement tag ;-)
